### ISSUE TITLE
[lightbox-media-viewer - with carousel]: QA: Screen reader is not identifying the slide headings

### ISSUE THREAD

**Issue by Gopi916** [carbon-design-system/carbon-for-ibm-dotcom#8911]
**Engineering info:**  
- Innovation Team engineer: 
- Innovation Team JIRA Ticket: https://jsw.ibm.com/browse/HC-2520
- DPO consulting engineer: N/A

-----

### Description

Page Element:
Headings in the carousel slide

Issue Description:
Screen reader does not identify the heading tag for the slide headings. they are announced only as text

Expected:
"Use Heading Level 1 for the main content of the page. Heading Level's should be in logical order, Starting from H1, H2, H3, H4, H5 and H6 respectively.
don't use Headings for font, For increasing font & 
style use CSS."
Do not skip Heading levels

User Impact:
If the screen reader does not identify page headings, users are unable to skim through various sections in the page and understand the content.

### Component(s) impacted

lightbox-media-viewer - with carousel

### Browser

Safari, Firefox

### Carbon for IBM.com version

canary

### Severity

Severity 3 = The problem is visible or noticeable to users but does not impede the usability or functionality. Affects minor functionality, has a workaround.

### Application/website

Carbon for IBM.com

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

https://ibmdotcom-webcomponents.s3.us-east.cloud-object-storage.appdomain.cloud/deploy-previews/8757/iframe.html?id=components-lightbox-media-viewer--with-carousel

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues

**Comment by gopinathan-n** [comment]
@andy-blum @kennylam Screen reader is announcing the headings in each slide now. Issue is fixed. Tested in Web components carousel default [https://ibmdotcom-webcomponents.s3.us-east.cloud-object-storage.appdomain.cloud/deploy-previews/9149/index.html?path=/story/overview-getting-started--page](url)
